rsc/client/valve/replay_test.go

```diff
@@ -0,0 +1,83 @@
+package valve
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDownload(t *testing.T) {
+	t.Run("download dota2 replay file should be success", func(t *testing.T) {
```

Comment:
why not use table-driven test here? at the very least, I would define `ctx` and `destination` at beginning of the function.

Reply:
> why not use table-driven test here?

because this is an integration test (call to external endpoint), I don't use table test here. Sometimes I use table test when it's unit test

Reply:
make sense, maybe also add separate build tag for integration test if you can.

Reply:
updated, i already add separate buld tag for integration test too